inference.Rmd

```diff
@@ -328,7 +342,7 @@ population_parameters
 ```
 The price per night of all Airbnb rentals in Vancouver, BC 
```

Comment:
above this: 
<img width="460" alt="Screen Shot 2021-09-26 at 9 08 42 PM" src="https://user-images.githubusercontent.com/3838091/134844218-42ce73a5-8506-4e3e-8877-b189d6c9fb5f.png">